files/common/var/lib/delphix-platform/os-migration

```diff
@@ -75,12 +96,17 @@ function perform_migration() {
 	"$MIGRATE_CONFIG_SCRIPT" post-upgrade >>"$MIGRATE_CONFIG_LOG" 2>&1 ||
 		die "Failed migrate_config post-upgrade, see" \
 			"$MIGRATE_CONFIG_LOG"
+	if zfs list rpool/fail-migration &>/dev/null; then
```

Comment:
would it be better to use the stress point script/architecture for this?

Reply:
Oh that was test code, it shouldn't have made it into the review, thanks for pointing that out. And yes indeed, I'm planning to add stress points here once they are supported for migration.